tests/test_deserialization.py

```diff
@@ -1067,6 +1067,24 @@ class MySchema(Schema):
         assert 'bar' not in data
         assert not errors
 
+    def test_partial_fields_deserialization(self):
+        class MySchema(Schema):
+            foo = fields.Field(required=True)
+            bar = fields.Field(required=True)
+            baz = fields.Field(required=True)
+
+        data, errors = MySchema().load({'foo': 3}, partial=tuple())
+        assert data['foo'] == 3
+        assert 'bar' in errors
+        assert 'baz' in errors
+
+        data, errors = MySchema().load({'foo': 3}, partial=('bar', 'baz'))
+        assert data['foo'] == 3
+        assert 'bar' not in data
+        assert 'baz' not in data
+        assert not errors
+
```

Comment:
Could you also add a test for overriding the `partial` attribute in `load`?

e.g.

```python
schema = MySchema(partial=True)
schema.load(.... , partial=('foo', ))
```